cloud/google/machineactuator.go

```diff
@@ -195,12 +199,12 @@ func (gce *GCEClient) CreateMachineController(cluster *clusterv1.Cluster, initia
 }
 
 func (gce *GCEClient) ProvisionClusterDependencies(cluster *clusterv1.Cluster, initialMachines []*clusterv1.Machine) error {
```

Comment:
`initialMachines` is not used anywhere inside `ProvisionClusterDependencies`. Can be removed as well.

Reply:
removed